.ci/azure-pipelines/build-windows.yml

```diff
@@ -18,22 +18,25 @@ jobs:
       BUILD_DIR: '$(Agent.WorkFolder)\build'
       VCVARSALL: '%ProgramFiles(x86)%\Microsoft Visual Studio\2017\Enterprise\VC\Auxiliary\Build\vcvarsall.bat'
       CONFIGURATION: 'Release'
+      VCPKG_ROOT: 'C:\vcpkg'
     steps:
       - script: |
-          vcpkg.exe install boost-system boost-filesystem boost-thread boost-date-time boost-iostreams boost-chrono boost-asio boost-dynamic-bitset boost-foreach boost-graph boost-interprocess boost-multi-array boost-ptr-container boost-random boost-signals2 eigen3 flann gtest qhull --triplet %PLATFORM%-windows && vcpkg.exe list
+          vcpkg.exe install eigen3 flann gtest qhull --triplet %PLATFORM%-windows && vcpkg.exe list
         displayName: 'Install Dependencies'
       - script: |
           rmdir %VCPKG_ROOT%\downloads /S /Q
           rmdir %VCPKG_ROOT%\packages /S /Q
         displayName: 'Free Up Space'
       - script: |
           call "%VCVARSALL%" %ARCHITECTURE%
-          set PATH=%VCPKG_ROOT%\installed\%PLATFORM%-windows\bin;%PATH%
+          set PATH=%VCPKG_ROOT%\installed\%PLATFORM%-windows\bin;C:\Program Files\Boost\1.69.0\lib;%PATH%
```

Comment:
I think the environment variable <code>BOOST_ROOT</code> can be used in here.
https://github.com/Microsoft/azure-pipelines-image-generation/blob/master/images/win/scripts/Installers/Install-Boost.ps1#L36-L37